<launch>
    <param name="cv_cap_prop_fps" type="double" value="30.0" />		
    <param name="rescale_camera_info" type="bool" value="false" />		
    <param name="camera_info_url" type="string" value="$(find gmsl_camera)/cfg/camera.yaml" />		
    <param name="imx390_cfg_url" type="string" value="$(find gmsl_camera)/cfg/stitcher-imx390cfg.yaml" />	
    
    <node pkg="gmsl_camera" type="gmsl_camera_node" name="gmsl_camera" output = "screen" respawn="true" />
</launch>
